mods/3d_armor/shields/init.lua

```diff
@@ -1,24 +1,65 @@
+-- support for i18n
+local S = armor_i18n.gettext
+
+local disable_sounds = minetest.settings:get_bool("shields_disable_sounds")
+local use_moreores = minetest.get_modpath("moreores")
+local function play_sound_effect(player, name)
+	if not disable_sounds and player then
+		local pos = player:get_pos()
+		if pos then
+			minetest.sound_play(name, {
+				pos = pos,
+				max_hear_distance = 10,
+				gain = 0.5,
+			})
+		end
+	end
+end
+
+if minetest.global_exists("armor") and armor.elements then
+	table.insert(armor.elements, "shield")
+	local mult = armor.config.level_multiplier or 1
+	armor.config.level_multiplier = mult * 0.9
+end
+
 -- Regisiter Shields
 
-minetest.register_tool("shields:shield_admin", {
-	description = "Admin Shield",
+armor:register_armor("shields:shield_admin", {
+	description = S("Admin Shield"),
 	inventory_image = "shields_inv_shield_admin.png",
-	groups = {armor_shield=1000, armor_heal=100, armor_use=0},
-	wear = 0,
+	groups = {armor_shield=1000, armor_heal=100, armor_use=0, not_in_creative_inventory=1},
 })
 
-if ARMOR_MATERIALS.wood then
-	minetest.register_tool("shields:shield_wood", {
-		description = "Wooden Shield",
+minetest.register_alias("adminshield", "shields:shield_admin")
+
+if armor.materials.wood then
+	armor:register_armor("shields:shield_wood", {
+		description = S("Wooden Shield"),
 		inventory_image = "shields_inv_shield_wood.png",
-		groups = {armor_shield=3, armor_heal=0, armor_use=2000},
-		wear = 0,
+		groups = {armor_shield=1, armor_heal=0, armor_use=2000, flammable=1},
```

Comment:
The stats here are different. This is also true for the other shields. We should keep them the same (and the flammable group isn't useful because there is no fire on MFF)

Reply:
Sorry for this, this wasn't intentional; it changed when I updated the whole modpack.
Working on it.